packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -102,15 +97,20 @@ export const DashboardReportTabView = ({ entityCode, TabSelector }) => {
     }
   }, [isScrolledPastTop, stickyBarsHeight]);
 
-  useEffect(() => {
-    // unset the selected dashboard when the data changes
-    // in case the selected one doesn't exist in the new data
-    setSelectedDashboard(false);
+  return {
+    scrollToTop,
+    topRef,
+    isScrolledPastTop,
+    onLoadTabBar,
+  };
+};
 
-    if (data) {
-      setDefaultDashboard(data, setSelectedDashboard);
-    }
-  }, [data, setSelectedDashboard]);
+export const DashboardReportTabView = ({ entityCode, TabSelector }) => {
+  const [selectedYear, setSelectedYear] = useUrlSearchParam('year', DEFAULT_DATA_YEAR);
+  const [selectedDashboard, setSelectedDashboard] = useUrlSearchParam('dashboard');
```

Comment:
@reviewer I thought about adding the url params as constants somewhere but decided that the cons outweighed the pros mostly because each one is only used twice. I am open to a suggestion to change that though.

Reply:
All good :+1: